assignment/scripts/array-practice.js

```diff
@@ -1,94 +1,113 @@
-console.log('****** Array Practice *******');
+// console.log('****** Array Practice *******');
 
-// 1. Creating arrays
-console.log('--- 1. Practice creating arrays ---');
+// // 1. Creating arrays
+// console.log('--- 1. Practice creating arrays ---');
 
-// Example: Array of animals
+// // Example: Array of animals
 let animalArray = ['fish', 'cat', 'bird', 'dog'];
-console.log('Animals are: ', animalArray);
+// console.log('Animals are: ', animalArray);
 
-// 1.a. TODO: Make an array with some favorite foods
+// // 1.a. TODO: Make an array with some favorite foods
+let favoriteFoods = ['Cheeseburger', 'Lava Cakes', 'Pizza', "Chocolate Strawberries"];
 
+// // 1.b. TODO: Log your array of foods to the console with a message, similar 
+// //      to the example above
 
-// 1.b. TODO: Log your array of foods to the console with a message, similar 
-//      to the example above
+console.log('My Favorite Foods are: ', favoriteFoods);
 
+// // 2. Array.length - tells you how many items are in the array
+// console.log('--- 2. Length of an array ---');
+// // Example: How many animals are in the array? 
+// console.log('Number of animals:', animalArray.length);
 
+// // 2.a. TODO: Log to the console the number of foods in your array
+console.log('The number of favorite foods: ', favoriteFoods.length);
 
-// 2. Array.length - tells you how many items are in the array
-console.log('--- 2. Length of an array ---');
 
-// Example: How many animals are in the array? 
-console.log('Number of animals:', animalArray.length);
+// // 3. Accessing array items
+// console.log('--- 3. Accessing items in an array ---');
 
-// 2.a. TODO: Log to the console the number of foods in your array
+// // Example: Log the first animal from the array using it's array index
+// console.log('First animal is', animalArray[0])
 
+// // 3.a. TODO: Log the second animal in the array 
+console.log('The second animal is: ', animalArray[1]);
 
-// 3. Accessing array items
-console.log('--- 3. Accessing items in an array ---');
+// // 3.b. TODO: Log the last animal in the array using it's array index 
+console.log('The last animal is: ', animalArray[3]);
 
-// Example: Log the first animal from the array using it's array index
-console.log('First animal is', animalArray[0])
 
-// 3.a. TODO: Log the second animal in the array 
+// // 3.c. (STRETCH) TODO: Log the last animal by using the array length, 
+// //      instead of the exact index number of the last item
+console.log(animalArray[animalArray.length-1]);
```

Comment:
Usually folks write operators like `-` with spaces around them.
```suggestion
console.log(animalArray[animalArray.length - 1]);
```